sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -29,6 +29,7 @@
 from apache_beam.io.filesystems import FileSystems
 from apache_beam.ml.inference.api import PredictionResult
 from apache_beam.ml.inference.base import ModelHandler
+from apache_beam.utils.annotations import experimental
 
 
 class PytorchModelHandler(ModelHandler[torch.Tensor,
```

Comment:
Is the intention to still keep `PytorchModelHandler`?

Reply:
No I think we want to drop `PytorchModelHandler`